waffle/models.py

```diff
@@ -243,6 +243,19 @@ def _is_active_for_language(self, request):
 
     def is_active(self, request):
         if not self.pk:
+            if get_setting('CREATE_MISSING_FLAGS'):
+                try:
+                    with transaction.atomic():
+                        get_waffle_flag_model().objects.create(
+                            name=self.name,
+                            everyone=get_setting('FLAG_DEFAULT')
+                        )
+                except IntegrityError:
```

Comment:
What would cause these errors? If such an error occurs, I would rather see it bubble up that be squashed with an uninformative log line.

Reply:
The IntegrityError would be thrown if you attempt to create a flag name that already exists, which can happen if two processes both encounter a flag that hasn't been created and attempt to create it. I caught it because I my philosophy around this change set was very "first do no harm" since it's intended to be a convenience and the behavior of returning the FLAG_DEFAULT is unaffected by whether or not the flag could be created I thought it was best to catch it.

Still, you're the maintainer so if you don't like that explanation I can definitely change it.

Reply:
Log as a warning, but include the stack trace by passing `exc_info=True` in your call to `warning`. Do that in all of the new exception handlers